### ISSUE TITLE
Screen reader is announcing incorrect role as toggle button for 'Submit Button.'

### ISSUE THREAD

**Issue by Kirank915** [microsoft/BotFramework-WebChat#5013]
### Is it an issue related to Adaptive Cards?

No.

### What is the PWD impact?

Users who rely on screen readers are getting impacted if NVDA is announcing incorrect role as toggle button for 'Submit Button.'

### What browsers and screen readers do this issue affect?

Others

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

ARIA roles

### What is the public URL for the website?

https://airtuserexpdev.azurewebsites.net/Dashboard 

### How to reproduce the issue?

Step 1: Open https://airtuserexpdev.azurewebsites.net/Dashboard  in Edge Browser->SAGE chat window will display.
Step 2: Type “What is the accessibility process”->Invoke “Dislike” button.
Step 3: Verify whether NVDA is announcing incorrect role as toggle button for 'Submit Button' or not.

### What do you expect?

NVDA should announce correct role for 'Submit Button.'
Ensure: NVDA should announce as "Submit button".

### What actually happened?

NVDA is announcing incorrect role as toggle button for 'Submit Button.'
Observation: NVDA is announcing as "Submit toggle button not pressed".

### Do you have any screenshots or recordings to repro the issue?

![MAS 4 1 2 - NVDA is announcing incorrect role as toggle button for 'Submit Button '](https://github.com/microsoft/BotFramework-WebChat/assets/117266902/a532937e-a982-4ff8-9906-ad746c7454b0)

### WCAG reference

MAS 4.1.2 – Name, Role, Value

**Comment by Kirank915** [comment]
This bug is copied from Accenture team: [BUG 11645870](https://microsoftit.visualstudio.com/OneITVSO/_workitems/edit/11645870)

**Comment by Kirank915** [comment]
#A11yMAS;#A11ySev2;#MAS1.3.1;#HCL;#Accessibility;#FeatureBotframeworkwebchat-Apr23;

**Comment by compulim** [comment]
This is related to Adaptive Cards.

All buttons in Adaptive Cards support repetitive submission. In nature, they are push button than an action button. Push buttons allows repetitive submissions.

If this is not desirable, please use technologies other than Adaptive Cards.

**Comment by Kirank915** [comment]
As per the above comment, this issue is owned by adaptive cards and not under bot framework team. Hence closed the bug and patner team (WWL) also changed the ownership in bugs.
